.prettierrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all",
```

Comment:
Not a fan of this one, I have to say. I think trailing commas are pure laziness and look rubbish.

Reply:
I'd have to agree with @paul-graham on this one.